android/TriblerApp/app/src/main/java/org/kamranzafar/jtar/TarHeader.java

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright 2012 Kamran Zafar 
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0 
+ * 
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * 
+ */
+
+package org.kamranzafar.jtar;

```

Comment:
rather than importing the code for a whole library as part of your sources, it may be more practical to have the build file take from an the jcenter and include it in your `build.gradle`

https://bintray.com/bintray/jcenter/org.kamranzafar%3Ajtar